firebase_admin/_auth_utils.py

```diff
@@ -275,6 +277,131 @@ def validate_provider_ids(provider_ids, required=False):
         validate_provider_id(provider_id, True)
     return provider_ids
 
+
+def validate_mfa_config(mfa_config):
+    """Validates the specified multi factor configuration."""
+
+    mfa_config_payload = {}
+    # validate multiFactorConfig
+    if not isinstance(mfa_config, dict):
```

Comment:
Should this check for MultiFactorConfig instead of dict?

Reply:
Can't do that since we're passing mfa as a dict object of type Any. 
The create_tenant() function does not take strong types for input. All the paramters are of type 'Any'.

Reply:
But we can do something like _auth_utils.validate_mfa_config(MultiFactorConfig(jsonData)) right?

Reply:
Ack